.github/workflows/deploy.yml

```diff
@@ -0,0 +1,65 @@
+# Sample workflow for building and deploying a VitePress site to GitHub Pages
+#
+name: Deploy VitePress site to Pages
+
+on:
+  # Runs on pushes targeting the `main` branch. Change this to `master` if you're
+  # using the `master` branch as the default branch.
+  push:
+    branches: [main]
```

Comment:
I created a `gh-pages` branch, we can merge this into `gh-pages` instead of the main branch.

Should we need to change branches here?

Reply:
Hi,

I just changed it.

Reply:
Done I just pushed the change

Reply:
Thank you for updating quickly : )

What do you think of just remaining documents files and removing other files?

Reply:
I'm not sure I think it's fine bcz doc files are in docs folder I think we should keep the other files. and Maybi just change the default github branch to that branch?

Reply:
OK, we can merge it first and give it a shot : )

Reply:
Encountered error:
https://github.com/neighborhood999/vue-signature-pad/actions/runs/5971534977/job/16200724902

Reply:
Let me check Give me few hours I will check bcz I tried in my doc repo there it's working.

Reply:
I think it's causing issue with the package.json I think it's better to just keep the doc and remove the files as you said

Reply:
@neighborhood999 

https://github.com/neighborhood999/vue-signature-pad/pull/395/files

I have removed other files